tests/test_login.py

```diff
@@ -1,19 +1,34 @@
-import unittest
 
 
-from trytond.tests.test_tryton import ModuleTestCase
-from trytond.tests.test_tryton import suite as test_suite
+from trytond.tests.test_tryton import DB_NAME
+from werkzeug.test import Client
+from trytond.wsgi import app
+from werkzeug.wrappers import Response
+from werkzeug.wrappers.json import JSONMixin, _JSONModule
 
 
-class AuthenticationDummyTestCase(ModuleTestCase):
-    'Test Authentication Dummy module'
-    module = 'authentication_dummy'
+class JSONResponse(JSONMixin, Response):
+    json_module = _JSONModule
 
-def test_login_dummy(self):
-    self.assertTrue(False)
 
-def suite():
-    suite = test_suite()
-    suite.addTests(unittest.TestLoader().loadTestsFromTestCase(
-            AuthenticationDummyTestCase))
-    return suite
+class LoginTestCase(object):
+
+    def test_login_dummy(self):
+
+        # GIVEN
+        client = Client(app, JSONResponse)
+
+        # WHEN
+        user_id, user_login = self.user
+        url = '/{}/'.format(DB_NAME)
+        result = client.post(
```

Comment:
Since this module will not register user applications nor any other kind of route nor middleware in trytond's wsgi application, I feel all this custom-client definition is an overkill for the purpose of the test.

I'd rather directly test the res.user get_login classmethod just like core's test suite does https://github.com/tryton/trytond/blob/39d651cad761277eae18cd0b52701eadf4d63669/trytond/tests/test_user.py#L71 and assume the RPC routing will do its work.